requirements.in

```diff
@@ -0,0 +1,9 @@
+pylint
+pytest
+pytest-mock
+black
+pip-tools
+pre-commit
+flake8
+pytest-mock
+coverage
```

Comment:
```suggestion
```

We won't want to add these development dependencies into our main `requirements.txt`. These are already defined in our `requirements-dev.txt`, which is relevant to contributors specifically.